### PR TITLE
Lesson 34

### PR DESCRIPTION
**Pull request by VladSmaug**

### CODE REVIEW

**Review by VladSmaug — COMMENTED** [review]
Can`t understand why after input at profile page some text in textarea and clicking add post button, text area field is not blanc

File: src/components/Profile/MyPost/index.jsx [hunk 1]
```diff
@@ -2,31 +2,37 @@ import { createRef } from "react";
...
+    updateNewPostText(text);
+  };
 
-const addPostHandler = () => {
-  const text = areaRef.current.value;
-  console.log(text);
-  addPost(text);
-};
+  const addPostHandler = () => {
+    addPost();
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Profile/MyPost/index.jsx:18]: If I`m not mistaken "addPost" should receive some props like text or some string, how does this function is working without passing any value?

**Comment by Bohdan-Zavhorodskyi** [comment]
> Can`t understand why after input at profile page some text in textarea and clicking add post button, text area field is not blanc

Have you tried to reset the value in the areaRef? `areaRef.current.value = ''`

File: src/components/Profile/MyPost/index.jsx (hunk 1 above)
Comment (VladSmaug) [reply src/components/Profile/MyPost/index.jsx:18]: This property is taken from the state

**Comment by VladSmaug** [comment]
> > Can`t understand why after input at profile page some text in textarea and clicking add post button, text area field is not blanc
> 
> Have you tried to reset the value in the areaRef? `areaRef.current.value = ''`

Thanks that`s actually works

**Review by Bohdan-Zavhorodskyi — APPROVED** [review]

**Pull request merged**